code/modules/research/designs/autolathe_designs.dm

```diff
@@ -798,10 +798,10 @@
 	build_path = /obj/item/weaponcrafting/receiver
 	category = list("hacked", "Security")
 
-/datum/design/c38
+/datum/design/c38_surplus
 	name = "Ammo Box (.38 surplus)"
-	id = "c38"
-	build_type = AUTOLATHE | PROTOLATHE
```

Comment:
Stealth prolathe nerf - add this to the PR description

Reply:
Added.

I'll also note #2775 already touched on this and restricted 38 ammo to the security protolathe only. No protolathe is also currently able to print surplus ammo either.

Reply:
Turns out Im insane and missed the fact the other surplus ammo types also have protolathe print flags. Readded the flag for consistency.